packages/components/src/components/hds/table/index.hbs

```diff
@@ -13,7 +13,9 @@
   <thead class="hds-table__thead">
     {{#if @columns}}
       <Hds::Table::Tr
+        @sortBySelectedOrder={{if (eq this.sortBy @selectableColumnKey) this.sortOrder}}
         @selectionScope="col"
+        @onClickSortBySelected={{if @selectableColumnKey (fn this.setSortBy @selectableColumnKey)}}
```

Comment:
nit: should we align these with what we have further down the file in the tr section?
```suggestion
        @selectionScope="col"
        @onClickSortBySelected={{if @selectableColumnKey (fn this.setSortBy @selectableColumnKey)}}
        @sortBySelectedOrder={{if (eq this.sortBy @selectableColumnKey) this.sortOrder}}
```